build-logic/src/main/kotlin/org/jetbrains/conventions/maven-publish.gradle.kts

```diff
@@ -1,10 +1,13 @@
 package org.jetbrains.conventions
 
+import org.gradle.internal.component.external.model.TestFixturesSupport.TEST_FIXTURE_SOURCESET_NAME
+
 plugins {
     id("org.jetbrains.conventions.base")
     `maven-publish`
     signing
     id("org.jetbrains.conventions.dokka")
+    id("dev.adamko.kotlin.binary-compatibility-validator")
```

Comment:
Rather than BCV-classic's approach of validating _all_ projects by default, and opting out on a case-by-case basis, I thought it made sense to apply BCV-MU in a slightly more targeted way and apply it to all published subprojects.